net/acme/files/run.sh

```diff
@@ -129,6 +129,12 @@ pre_checks()
 
 post_checks()
 {
+    if [ -f "$STATE_DIR/post-acme.user" ]; then
+        log "Running user-provided cleanup script from $STATE_DIR/post-acme.user."
+        . "$STATE_DIR/post-acme.user" || return 1
+        return 0
```

Comment:
Not sure if it's a good idea to just unconditionally override the default behaviour. There are certainly cases where a post script should be *in addition* to the default behaviour, and with this the user would have to reimplement the defaults themselves.

Reply:
You could be right - I've updated the PR with commits to change the behaviour for the cleanup script to be run in addition to the built-in behaviour.